examples/keygen.php

```diff
@@ -1,20 +1,24 @@
 <?php
 
+use phpseclib3\Crypt\RSA;
+use phpseclib3\Crypt\RSA\Formats\Keys\PKCS1;
+
 @include_once dirname(__FILE__).'/../vendor/autoload.php';
 require_once dirname(__FILE__).'/../lib/openpgp.php';
 require_once dirname(__FILE__).'/../lib/openpgp_crypt_rsa.php';
 
-$rsa = new \phpseclib\Crypt\RSA();
-$k = $rsa->createKey(512);
-$rsa->loadKey($k['privatekey']);
+$privateKey = RSA::createKey(512);
+$publickey = $privateKey->getPublicKey();
```

Comment:
should probably pick capital K or not between these two?

Reply:
sorry i'm not clear what you want me to do here :)